### ISSUE TITLE
Parameter setting for cifar10

### ISSUE THREAD

**Issue by ShiningSord** [LiJunnan1992/DivideMix#9]
Hi!
I trained on cifar10 dataset with 40% asymmetric noise on default parameter setting. and I got only 83.5% accuracy on test set. 
I noticed that sentence 'We choose λu from {0, 25, 50, 150} using a small validation set.' in your paper. So how to choose λu for different noise mode and radio to get the best accuracy?
Thank you very much!
